src/components/InsertTextDialog.vue

```diff
@@ -34,6 +36,11 @@ export default {
     methods: {
         onSave() {
             console.log("Saved data for Edit Device");
+        },
+        onclick() {
+            Registry.viewManager.activateTool("InsertTextTool");
+            insertTextTool.text(document.getElementById("inserttext_textinput").value);
```

Comment:
You can get the data via data binding, however, this will change soon when one of the open-source contributors will add the new version of the dialog